client/app/models/contact.coffee

```diff
@@ -84,7 +87,7 @@ module.exports = class Contact extends Backbone.Model
             attrs.url = mainUrl.value
 
         options.attrs = attrs
```

Comment:
If we modify directly the passed `attrs` arg, I suppose we don't need to add it in `options`?